plugin-tester-java/src/main/proto/stateful-helloworld.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "example.myapp.statefulhelloworld.grpc";
+option java_outer_classname = "HelloWorldProto";
+
+package statefulhelloworld;
+
+//#services
+// The greeting service definition.
+service GreeterService {
+    // Sends a greeting
+    rpc SayHello (HelloRequest) returns (HelloReply) {}
+
+    // changes the used greeting
+    rpc ChangeGreeting (ChangeRequest) returns (ChangeResponse) {}
```

Comment:
Should we also show streaming?

Reply:
I was thinking about that, but let's keep it simple like this to start with.